project-management/1-1-meetings/Sophia-Batchelor/weekly-meetings-2024.md

```diff
@@ -17,6 +18,44 @@
 * [29 January 2024](#29-January-2024)
 * [23 January 2024](#23-January-2024)
 
+# 1 July 2024
+
+## Project Updates [[taskboard]](https://github.com/orgs/aim-rsf/projects/15/views/2)
+
+### Widening Participation 
+* No further updates beyond planning out activities for the students during RCM co-working 
+
+### PPIE Sessions 
+* Session this week
+* Have been trying to prepare questions and ways to stimulate conversation so that it's not stale.
+  *  Been thinking about asking people what they have enjoyed
+  *  Is this different from other/previous communities you've been a part of
+  *  What would you like to happen
+  *  What is needed for that to happen 
+
+### RCM
+* Gave PPIE updates to H1 reporting 
+* Still supporting the RCM paper (with case studies)
+
+### What would you like to work on in the week ahead?
+* Lots of events this week which will fill up the time
+  * ECR day
+  * PPIE session
+  * Collaboration Cafe
+  * AutSPACEs (would love to sync up about the chapter 
+
+* Cezanne check calendar
```

Comment:
Not 100% what is meant here? :)

Reply:
about the co-creation chapter work. Mostly reading through the wonderful work and asking if there's any support/review that would be helpful.

Reply:
I meant the "Cezanne check calendar", that seems distinct from the collab cafe? 🤔

Reply:
Ah I think it was that I hadn't put in a sick day and needed to check the dates and submit the absence :)